huxley/core/models.py

```diff
@@ -361,6 +361,20 @@ def remove(assignment_data):
 
         return failed_assignments
 
+    @classmethod
+    def update_assignment(cls, **kwargs):
+        assignment = kwargs['instance']
+        # Check that this assignment already exists
+        if assignment.id:
+            old_assignment = cls.objects.get(id=assignment.id)
+            # Check if school changed
+            if not assignment.school.id == old_assignment.school.id:
+                assignment.rejected = False
+                delegates = Delegate.objects.filter(assignment_id=old_assignment.id)
```

Comment:
You can do this more efficiently (and tersely) with the `update` method of a QuerySet, I believe. However, double-check that there aren't any `pre_save`/`post_save` signals attached to Delegate, because `QuerySet.update` doesn't trigger them.

https://docs.djangoproject.com/en/1.8/ref/models/querysets/#update